AK/Error.h

```diff
@@ -22,35 +23,51 @@ namespace AK {
 
 class Error {
 public:
-    static Error from_errno(int code) { return Error(code); }
-    static Error from_syscall(StringView syscall_name, int rc) { return Error(syscall_name, rc); }
-    static Error from_string_literal(StringView string_literal) { return Error(string_literal); }
+    static Error from_errno(int code, SourceLocation location = SourceLocation::current())
```

Comment:
What about the `ErrorOr` constructor below, we should do the same there as well?

Reply:
Seems I didn't see them, but you're right. Added them to the patch. :^)